sqlmesh/migrations/v0001_init.py

```diff
@@ -0,0 +1,51 @@
+"""All migrations should be named _XXXX.py, they will be executed sequentially.
+
+If a migration alters the payload of any pydantic models, you should not actually use them because
+the running model may not be able to load them. Make sure that these migration files are standalone.
+"""
+from sqlglot import exp
+
+
+def migrate(state_sync):  # type: ignore
```

Comment:
Don't we want to add a signature with imports under `t.TYPE_CHECKING`?

Reply:
decided against it